src/Collector/SuperGlobalsCollector.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SensioLabs\Deptrac\Collector;
+
+use SensioLabs\Deptrac\AstRunner\AstMap;
+use SensioLabs\Deptrac\AstRunner\AstMap\AstClassReference;
+
+final class SuperGlobalsCollector implements CollectorInterface
+{
+    public function getType(): string
+    {
+        return 'superGlobals';
```

Comment:
I would have choosen a different name (e.g. `super-globals` but since other collectors use camel-case its fine like this for consistency)

Reply:
I don't want to start bikeshedding, but you could use `superglobals` as this would be closer to the PHP docs: https://www.php.net/manual/en/language.variables.superglobals.php